tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -28,8 +44,27 @@ def self.run(params)
 
         parse_dsym_paths(params[:dsym_path]).each do |dsym_path|
           if dsym_path.end_with?(".zip") or File.directory?(dsym_path)
-            args = upload_args(dsym_path, params[:symbol_maps_path], params[:upload_url], params[:project_root], api_key, verbose, params[:ignore_missing_dwarf], params[:ignore_empty_dsym])
-            success = Kernel.system(UPLOAD_SCRIPT_PATH, *args)
+            args = upload_args(
+              "\"#{dsym_path}\"",
+              params[:upload_url],
+              params[:project_root],
+              api_key,
+              verbose,
+              params[:ignore_missing_dwarf],
+              params[:ignore_empty_dsym],
+              params[:dryrun],
+              params[:log_level],
+              params[:port],
+              params[:retries],
+              params[:timeout],
+              params[:configuration],
+              params[:scheme],
+              params[:config_file],
+              params[:xcode_project]
+            )
+            bugsnag_cli_command = "#{bugsnag_cli_path} upload dsym #{args.join(' ')}"
+            UI.message("Running command: #{bugsnag_cli_command}")
```

Comment:
What tools do we have in the way of log level? I feel like the CLI version and command are really only useful for debugging.